ecf/rtofs/plots/jevs_rtofs_plots.ecf

```diff
@@ -35,11 +37,12 @@ export STEP=plots
 export COMPONENT=rtofs
 
 # set up VDATE and COMIN and COMOUT
-export VDATE=$(date --date="2 days ago" +%Y%m%d)
+export VDATE=$(date --date="4 days ago" +%Y%m%d)
 
-export COMIN=${COMIN:-$(compath.py $envir/com/$NET/${evs_ver})}
-export COMOUT=/lfs/h2/emc/ptmp/$USER/$NET/${evs_ver}
-export COMOUTfinal=/lfs/h2/emc/vpppg/noscrub/$USER/stat_archive/RTOFS
+#export COMIN=${COMIN:-$(compath.py $envir/com/$NET/${evs_ver})}
```

Comment:
This line (42) should be deleted. That construct is not useful in the ecf file since there is no processing upstream. This line should be in the job file, and it is already.

Reply:
Done